main/src/main/java/org/mobilitydata/gtfsvalidator/notice/BlockTripsWithOverlappingStopTimesNotice.java

```diff
@@ -3,7 +3,11 @@
 import com.google.common.collect.ImmutableMap;
 import org.mobilitydata.gtfsvalidator.type.GtfsDate;
 
-/** Describes two trips with the same block id that have overlapping stop times. */
+/**
+ * Describes two trips with the same block id that have overlapping stop times.
+ *
+ * <p>Severity: {@code SeverityLevel.WARNING}
```

Comment:
The spec doesn't explicitly say you can't do this, although it logically doesn't make sense. So for the canonical validator, I believe this should still be a WARNING?

@MobilityData/transit-specs?

Also, this is listed as `BlockTripsWithOverlappingStopTimes` (no `Notice` on the end) in the RULES.md table.

Reply:
This is a data bug, that must be an ERROR.

I would like to repeat: we should not be prescriptionists. Forget the GTFS spec. It was written ages ago and it not describing the real feeds. The source of truth are the existing use cases. We may and we should update GTFS reference when somethings that it says does not make sense.

Reply:
I agree with @aababilov on the need to follow the current data that is already live in the field. Problematic -negative- habits could be rooted out in a spec update indeed later down the line.
That being said, some kind of process is then needed to know what's in the current data, I certainly don't have much expertise myself in that regard. I'll be watching the conversation 👀

Reply:
I agree, it doesn't logically make sense. This can be treated as an ERROR.
Registered in Discrepancies Specification vs Validator https://github.com/MobilityData/gtfs-tasks/issues/87